ui/src/pages/publications/[id]/flag/[flagId]/index.tsx

```diff
@@ -0,0 +1,309 @@
+import React from 'react';
+import { NextPage } from 'next';
```

Comment:
This should probably follow a similar convention to React (`import * as Next from 'next'`) given our preference now to avoid destructuring.

Reply:
Just fixed in the latest commit